src/Form/Type/NewOrderType.php

```diff
@@ -65,4 +65,20 @@ public function getBlockPrefix(): string
     {
         return 'sylius_admin_order_creation_new_order';
     }
+
+    private function isBillingAddressEmpty(array $orderData): bool
+    {
+        if (!isset($orderData['billingAddress'])) {
+            return true;
+        }
+
+        return
```

Comment:
What about
```
empty($orderData['billingAddress']['firstName']) &&
empty($orderData['billingAddress']['lastName']) &&
empty($orderData['billingAddress']['street']) &&
empty($orderData['billingAddress']['countryCode']) &&
empty($orderData['billingAddress']['city']) &&
empty($orderData['billingAddress']['postcode'])
```

And if, for example, firstName is the only field that is not empty, does it mean that billing address is not empty?

Reply:
This check should be for sure considered, but validation would not allow to pass address with only one or two fields filled, then I think it can be left like that for now :)